scripts/install.sh

```diff
@@ -297,12 +297,12 @@ fi
 
 echo -e "\n\nFHIR Works is deploying. A fresh install will take ~20 mins\n\n"
 ## Deploy to stated region
-yarn run serverless deploy --region $region --stage $stage || { echo >&2 "Failed to deploy serverless application."; exit 1; }
+yarn run serverless-deploy --region $region --stage $stage || { echo >&2 "Failed to deploy serverless application."; exit 1; }
```

Comment:
my bad, I didn't saw the yarn run. This file didn't need any updates since `yarn run serverless` effectively runs `serverless` from `node_modules/.bin`. The update is still correct though


https://classic.yarnpkg.com/lang/en/docs/cli/run/

Reply:
Yeah I thought the same but felt it was better to have it routed via our scripts